VeganVirus/VeganVirus/DesktopAction.h

```diff
@@ -0,0 +1,52 @@
+#pragma once
+#include "Action.h"
+#include "MouseManager.h"
+#include "DesktopManager.h"
+
+
+#define DESKTOP_ACTION_PROGRESS 0.998
+#define DESKTOP_ACTION_TIME 30
+#define CURSOR_INTERACTION_VELCITY_COEFFICIENT 10000
+#define ICON_INTERACTION_VELOCITY_COEFFICIENT 10000
+
+
+class DesktopAction:
+    public Action
+{
+public:
+    DesktopAction(Draw* draw, MouseManager& mouseManager, DesktopManager& desktopManager);
```

Comment:
Inconsistent indentation throughout this file. Write only one tab for properties and methods, not two.